services/cambio/getCambio.js

```diff
@@ -0,0 +1,8 @@
+const axios = require('axios');
+
+export const getCambio = async (data, moeda) => {
+  const resp = await axios.get(
+    `https://olinda.bcb.gov.br/olinda/servico/PTAX/versao/v1/odata/CotacaoMoedaDia(moeda=@moeda,dataCotacao=@dataCotacao)?@moeda='${moeda}'&@dataCotacao='${data}'&$top=100&$format=json`
+  );
+  return resp;
```

Comment:
Também sugiro alterar para algo assim e retornar apenas o dado necessário
```suggestion
  return resp.data;
```